app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -18,25 +20,33 @@ export default function NoteForm({ onClose, refetch, note, isEdit }) {
       refetch();
       onClose();
     } catch (err) {
-      Toastr.error(err);
+      logger.error(err);
```

Comment:
Changed to logger since we are already catching the error and displaying Toastr in the axios response handler.